manuscript/premix/ansible/operation.md

```diff
@@ -124,6 +124,18 @@ To deploy traefik (overlay), traefikv1, and traefik-forward-auth:
 ansible-playbook -i host deploy.yml -t traefik,traefikv1,traefik-forward-auth
 ```
 
+### Deploy (semi-autopilot)
+
+Deploying on full autopilot above installs _a lot_ of stuff (and more is being added every day). There's a good chance you don't want everything that is or will be included in the playbook. We've created a special tag that will install the base infrastructure up to a point that you can then choose which recipes to install with the "selective" deploy method described above.
+
+To deploy the base infrastructure:
+
+```
+ansible-playbook -i host deploy.yml -t infrastructure
```

Comment:
I'm only reading this in the context of the PR, but is `hosts` what we tell the user to name their own hosts file?

Reply:
The other instructions on the page say to use `hosts`, but I'm actually using `hosts.<username>` on my local. That's what we have in the `.gitignore` file as well.

Reply:
Updated throughout the doc to use the hosts.your-username scheme